cli.js

```diff
@@ -10,18 +10,33 @@ const graphqlHTTP = require('express-graphql')
 const OasGraph = require('./lib/index.js')
 const path = require('path')
 const request = require('request')
+const graphql = require('graphql')
 const fs = require('fs')
 
 const app = express()
 
+const outputFileName = 'schema.graphql';
+
 if (process.argv.length <= 2) {
-  const usage = ' <OAS JSON file path or remote url> [port number]'
-  console.log('Usage: ' + __filename + usage)
+  const usage = 'oasgraph <OAS JSON file path or remote url> [port number] [--save]'
+  console.log('Usage: ' + usage)
   process.exit(-1)
 }
 
 let filePath = process.argv[2]
-let portNumber = process.argv[3] ? process.argv[3] : 3001
+
+// default values if only the url or local file is specified
+let portNumber = 3001
+let saveSchema = false
+
+// examine the last two arguments
+for (var i=3; i<=process.argv.length-1; i++) {
```

Comment:
This may be a problem if a user adds more arguments then intended. It might be okay for now but could be hazardous if we want to add more options, refer to #44, What do you think @ErikWittern?

Reply:
I agree. I believe in a near future, we should integrate something like  https://www.npmjs.com/package/minimist to handle more complex command line options. I will place a MAX_PARAM constant for now. 
thoughts?

Reply:
Yes, I completely agree. Setting up the CLI interface from scratch will definitely lead to trouble. We should look into it if we want to build out the CLI tool more.

Reply:
ok. for now, if you inspect the code, the `--save` option is safe and now for port number the user can specify multiple argumetns if he wants, it will only take the ones that are numbers, so the user can enter:

`oasgraph openapi.json 3002 --save mickey pluto`

and still will work , and if user enters 

`oasgraph openapi.json 3002 --save mickey pluto 3003`

it will start on port number 3003 , any way, I think it is safe now 😄